test/integration/goldens/asset/AssetServiceSettings.java

```diff
@@ -54,7 +54,21 @@
  * <p>The builder of this class is recursive, so contained classes are themselves builders. When
  * build() is called, the tree of builders is called to create the complete settings object.
  *
- * <p>For example, to set the total timeout of exportAssets to 30 seconds:
+ * <p>For example, to set the total timeout of batchGetAssetsHistory to 30 seconds:
+ *
```

Comment:
Just for confirmation, the method change from `exportAssets ` to `batchGetAssetsHistory ` is intended. Same below. Thanks

Reply:
Yes. In the settings, we pick the first pure unary rpc as sample to show.